documentation/src/main/asciidoc/topics/proc_creating_server_security_realms.adoc

```diff
@@ -50,7 +50,13 @@ You must manually synchronize credentials in a properties realm to each node in
 . Use the `security-realms` element in the `security` configuration to contain create multiple security realms.
 . Add a security realm with the `security-realm` element and give it a unique name with the `name` attribute.
 +
-To follow the example, create one security realm named `application-realm` and another named `management-realm`.
+[IMPORTANT]
+====
+Do not add special characters, such as hyphens (`-`) or ampersands (`&`), to security realm names.  
+Special characters in security realm names can make them unusable by {brandname} Server endpoints.
```

Comment:
```suggestion
{brandname} Server endpoints can become unreachable if security realm names contain special characters.
```

Reply:
Hi @oraNod . Implemented your suggestion, changed the `name: "trust-store-realm"` entry in the YAML file, and then updated the commit message.